components/service/glean/src/main/java/mozilla/components/service/glean/ping/PingMaker.kt

```diff
@@ -29,8 +30,22 @@ internal class PingMaker(
      * @return a string containing the date and time.
      */
     private fun getISOTimeString(): String {
-        val dateFormat = SimpleDateFormat("yyyy-MM-dd'T'HH:mmXXX", Locale.US)
-        return dateFormat.format(Date())
+        val dateFormat = SimpleDateFormat("yyyy-MM-dd'T'HH:mmZ", Locale.US)
+        val timeString = StringBuilder(dateFormat.format(Date()))
+
+        // Due to limitations of SDK version 21, there isn't a way to properly output the time
```

Comment:
Can we document this in the `docs/pings.md`, when we talk about the fields using this date? We can create a new section in the doc, and reference it from each field.